tests/acceptance/expected-failures-API-on-OCIS-storage.md

```diff
@@ -577,31 +577,6 @@ cannot share a folder with create permission
 
 - [apiSharePublicLink2/uploadToPublicLinkShare.feature:198](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiSharePublicLink2/uploadToPublicLinkShare.feature#L198)
 
-#### [Resharing does not work with ocis storage](https://github.com/owncloud/product/issues/265)
-
-- [apiShareReshareToShares3/reShareWithExpiryDate.feature:403](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareReshareToShares3/reShareWithExpiryDate.feature#L403)
```

Comment:
Note: these were duplicated further down the expected-failures. I removed the duplication.